contracts/interfaces/ICover.sol

```diff
@@ -48,25 +87,11 @@ struct CoverData {
 struct CoverSegment {
   uint96 amount;
   uint32 start;
-  uint32 period;  // seconds
-  uint16 gracePeriodInDays;
+  uint32 period; // seconds
+  uint32 gracePeriod; // seconds
```

Comment:
i'd add a comment how many years this can actually store 

it's like 136 or so since epoch time.

Reply:
We're using uint32 for timestamps for a while now, and not just for (short) periods of time but for actual timestamps (dates). The comment would be redundant